api/src/test/java/org/motechproject/nms/api/osgi/KilkariControllerBundleIT.java

```diff
@@ -272,6 +279,25 @@ public void testCreateSubscriptionRequestDifferentPacks() throws IOException, In
     }
 
     @Test
+    public void testCreateSubscriptionsNoLanguageInDB() throws IOException, InterruptedException {
+        setupData();
```

Comment:
Line 238? :)

On a bigger note though, @before is probably a good standard to follow